room.go

```diff
@@ -72,6 +83,37 @@ func (r *Room) Draw(frame *Frame) {
 	}
 }
 
+// FindCallback returns the callback with the given name, or nil if not found.
+func (r *Room) FindCallback(name string) *ScriptCallback {
+	for _, cb := range r.callbacks {
+		if cb.Name == name {
+			return cb
+		}
+	}
+	return nil
+}
+
+// GetScriptField returns the script field with the given name, or nil if not found.
+func (r *Room) GetScriptField(name string) *ScriptEntityValue {
+	for _, obj := range r.objects {
+		if obj.Name == name {
```

Comment:
This check does not allow using the `Object.Name` field as a label in the game UI. This was causing the issue about the missing method, which was not reproducible in the example (since the object fields and their respective names are the same).

Reply:
Nice catch! For the record, we fixed this in #60 .